src/commands/kick/kick.ts

```diff
@@ -0,0 +1,61 @@
+import { Command, SafetyJim } from '../../safetyjim/safetyjim';
+import * as Discord from 'discord.js';
+
+class Kick implements Command {
+    public usage = 'kick - boot the naughty people <:';

```

Comment:
maybe something like this `kick @user <reason> - kicks the user with specified reason`